doc/opensprint/usage.md

```diff
@@ -0,0 +1,122 @@
+The `dist` client binary
+========================
+
+*Note: the binary name, like everything else, is subject to change.*
+
+The `dist` tool is the entrypoint for packaging and shipping content. It accepts the following subcommands that will each be detailed in the following sections.
+
+  - The lower-level `put`/`get` couple act **locally** on **opaque objects** identified by a **unique ID**
+  - The higher-level `push`/`pull` couple interact with a **remote registry** on **content** described through a **manifest**
+
+To be detailed:
+
+  - The `pack` command is used to generate and modify resource manifests
+  - The `list` subcommand lists locally available content
+  - The `remove` deletes locally available content
+  - The `login` subcommand authenticates to a remote registry
+
+## Definitions
+
+A **manifest** is a signed document identifying a particular named resource. It is comprised of sections which can be of different types. In a first version, we only expect to be dealing with Docker images, which manifests may only hold an "image" section (to be specified separately, but that we'll assume to define a list of layers).
+
+## Examples
+
+Creating and pushing a Docker image:
+
+```
+$> dist insert content1.tar
+# dbe80f010ab3c73df8f3f6a54e161eecdf561f4d
+$> dist insert content2.tar
```

Comment:
On Wed, Jan 07, 2015 at 09:53:19AM -0800, Andy Goldstein wrote:
> s/insert/put

I fixed these in icecrime/distribution#1 while adding a -t option.